dspace/config/crosswalks/oai/transformers/openaire4.xsl

```diff
@@ -0,0 +1,372 @@
+                <xsl:text>eng</xsl:text>
+            </xsl:when>
+            <xsl:when test="$lc_value = 'es'">
+                <xsl:text>spa</xsl:text>
+            </xsl:when>
+            <xsl:when test="$lc_value = 'de'">
+                <xsl:text>deu</xsl:text>
+            </xsl:when>
+            <xsl:when test="$lc_value = 'fr'">
+                <xsl:text>fra</xsl:text>
+            </xsl:when>
+            <xsl:when test="$lc_value = 'it'">
+                <xsl:text>ita</xsl:text>
+            </xsl:when>
+            <xsl:when test="$lc_value = 'ja'">
+                <xsl:text>jap</xsl:text>
+            </xsl:when>
+            <xsl:when test="$lc_value = 'zh'">
+                <xsl:text>zho</xsl:text>
+            </xsl:when>
+            <xsl:when test="$lc_value = 'pt'">
+                <xsl:text>por</xsl:text>
+            </xsl:when>
+            <xsl:when test="$lc_value = 'tr'">
+                <xsl:text>tur</xsl:text>
+            </xsl:when>
+            <xsl:otherwise>
+                <xsl:value-of select="."/>
+            </xsl:otherwise>
+        </xsl:choose>
+    </xsl:template>
+        
+    <!-- Modifying dc.rights -->
+    <!-- Removing unwanted -->
+    <xsl:template
+        match="/doc:metadata/doc:element[@name='dc']/doc:element[@name='rights']/doc:element/doc:element"/>
+    <!-- Replacing -->
+    <xsl:template
+        match="/doc:metadata/doc:element[@name='dc']/doc:element[@name='rights']/doc:element/doc:field/text()">
+        <xsl:variable name="value" select="."/>
+        <xsl:variable name="lc_value">
+            <xsl:call-template name="lowercase">
+                <xsl:with-param name="value" select="."/>
+            </xsl:call-template>
+        </xsl:variable>
+        <xsl:choose>
+            <xsl:when
+                test="$lc_value = 'open access' or $lc_value = 'openaccess' or $value = 'http://purl.org/coar/access_right/c_abf2'">
+                <xsl:text>open access</xsl:text>
+            </xsl:when>
+            <xsl:when
+                test="$lc_value = 'embargoed access' or $lc_value = 'embargoedaccess' or $value = 'http://purl.org/coar/access_right/c_f1cf'">
+                <xsl:text>embargoed access</xsl:text>
+            </xsl:when>
+            <xsl:when
+                test="$lc_value = 'restricted access' or $lc_value = 'restrictedaccess' or $value = 'http://purl.org/coar/access_right/c_16ec'">
+                <xsl:text>restricted access</xsl:text>
+            </xsl:when>
+            <xsl:when
+                test="$lc_value = 'metadata only access' or $lc_value = 'closedaccess' or $value = 'http://purl.org/coar/access_right/c_14cb'">
+                <xsl:text>metadata only access</xsl:text>
+            </xsl:when>
+            <xsl:otherwise>
+                <xsl:value-of select="$value"/>
+            </xsl:otherwise>
+        </xsl:choose>
+    </xsl:template>
+
+    <!-- Modifying and normalizing dc.type -->
```

Comment:
Again, it'd be good to note what "standard" format these are being normalized into.  That'll make it easier to add to or correct later on (as needed)